generated/InlineObject.h

```diff
@@ -0,0 +1,40 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_Inline_object_H_
```

Comment:
I don't know where `InlineObject.h` and `InlineObject1.h` come from, but I guess we should get rid of them.

Reply:
OK, these classes are generated from schemas that are not defined in the schemas section, but directly within the path where they are used. In this example, `InlineObject1.h` is the schema of the response body for a POST to '/pet/{petId}/uploadImage'.

Do we also have cases like this, or are all our objects defined in the schema section?

Reply:
It seems that in our use case, this does not occur. Maybe we can ignore them, or I try to find a way to turn them off.

Reply:
I think its fine to ignore that. If we happen to generate such objects, so be it. Its all header-only, so what we don't use, we don't ship.

Reply:
Just to follow up on this, we can also use `-DskipFormModel=true` to avoid generating these.